robotbase/src/test/java/frc/robot/DriveLockReefTests.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import frc.robot.commands.drive.DriveToReef;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class DriveLockReefTests {
```

Comment:
Can we add a test or two for when the X or Y deltas are the same between the two poses? e.g. (1, 0) + (5, 0) or (1, 1) + (1, 5)